src/shogun/lib/SGVector.h

```diff
@@ -474,6 +477,25 @@ template<class T> class SGVector : public SGReferencedData
 		 * @return matrix
 		 */
 		static void convert_to_matrix(T*& matrix, index_t nrows, index_t ncols, const T* vector, int32_t vlen, bool fortran_order);
+
+		template<class Archive>
```

Comment:
This seems ok but what we really need is a patch that serialises the parameter map of the tag framework....thoughts?